### PR TITLE
2 arrows mode (#146)

### PR DESCRIPTION
**Pull request by etiennehomer**
Signed-off-by: HOMER Etienne <etiennehomer@gmail.com>

### CODE REVIEW

File: src/components/network/line-layer.js [hunk 1]
```diff
@@ -70,7 +74,7 @@ class LineLayer extends CompositeLayer {
             compositeData = this.state.compositeData;
         }
 
-        if (changeFlags.dataChanged || (changeFlags.propsChanged && oldProps.lineFullPath !== props.lineFullPath)) {
+        if (changeFlags.dataChanged || (changeFlags.propsChanged && oldProps.lineFullPath !== props.lineFullPath)  || props.lineFlowMode != oldProps.lineFlowMode) {
```
Comment (geofjamg) [inline src/components/network/line-layer.js:80]: Should it be?
```js
(changeFlags.propsChanged && (oldProps.lineFullPath !== props.lineFullPath || props.lineFlowMode != oldProps.lineFlowMode))
```

File: src/components/network/line-layer.js [hunk 2]
```diff
@@ -122,12 +126,25 @@ class LineLayer extends CompositeLayer {
 
                     line.cumulativeDistances = lineData.cumulativeDistances;
                     line.positions = lineData.positions;
-                    return [...new Array(arrowCount).keys()].map(index => {
-                        return {
-                            distance: index / arrowCount,
+                    if(props.lineFlowMode != LineFlowMode.FEEDERS) {
```
Comment (geofjamg) [inline src/components/network/line-layer.js:132]: !==

File: src/components/network/line-layer.js (hunk 1 above)
Comment (etiennehomer) [reply src/components/network/line-layer.js:80]: Yes

File: src/components/network/line-layer.js [hunk 3]
```diff
@@ -11,11 +11,18 @@ import ArrowLayer, {ArrowDirection} from "./layers/arrow-layer";
 import getDistance from "geolib/es/getDistance";
 
 const DISTANCE_BETWEEN_ARROWS = 10000.0;
+//Constants for Feeders mode
+const START_ARROW_POSITION = 0.1;
+const END_ARROW_POSITION = 0.9;
+//Constants for voltage labels
```
Comment (geofjamg) [inline src/components/network/line-layer.js:17]: this is not voltage but flow

Comment (geofjamg) [reply src/components/network/line-layer.js:17]: You should name it START_LABEL_POSITION and END_LABEL_POSITION

**Pull request merged**